test/controllers/shared/basic_rest_response_test.rb

```diff
@@ -36,5 +36,121 @@ def basic_edit_test(object_found = nil)
         end
       end
     end
+
+    def basic_pagination_per_page_test
+      context 'GET #index' do
+        entries_per_page = Setting[:entries_per_page] ? Setting[:entries_per_page] : 5
```

Comment:
This should be in the setup block, if you need to access it in the tests make it an instance variable.